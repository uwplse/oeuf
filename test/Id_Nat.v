Require List SourceLang Pretty OeufPlugin.OeufPlugin.
Import List.ListNotations.

Eval compute Then Write To File "id_nat.oeuf" (Pretty.expr.print (@SourceLang.id_nat_reflect [])).
