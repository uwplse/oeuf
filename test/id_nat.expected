id_nat(3) = 3
